src/cmd/create.rs

```diff
@@ -86,8 +103,13 @@ impl Cmd {
 
 impl Basic {
     pub async fn run(&self, opts: Opts) -> Result {
-        let seed_words = if self.seed {
-            Some(get_seed_words()?)
+
+        if self.seed {
```

Comment:
maybe add another bail around here to guard against: ```self.seed_mobile && self.seed_bip39```?

another options is to make `--seed mobile/bip39` an enum perhaps. Is there a another seed type down the line like bip42?

Reply:
Ahh, so you mean rather than have distinct flags for each type like I do now, make `--seed` take an argument on the cmdline that is the type, keeping the current logic which does `get_seed_words` after the cmdline is parsed?  That would definitely age better if there are more types. 

If I can't make that work, I'll definitely add a guard both --seed-* flags being set.

Reply:
Yea, you can do it pretty easily with an enum type and the EnumString macro to do the "FromStr" parsing from the command line:

```
use strum_macros::EnumString;

#[derive(Debug, EnumString)]
pub enum Seed {
    mobile,
    bip39, 
}
```

Then in the Opt struct, you put:
```
    #[structopt(long)]    
    /// Use space separated seed words to generate a wallet from seed
    seed: Seed
```

Reply:
Ok, just force pushed changes to cmd/create.rs and cmd/mod.rs so that the UI is now
`--seed bip39|mobile` 

Thanks for the suggestions, @lthiery , I like this much better.